Build failure on Windows

```
C:\couchbase-lite-java-forestdb>gradlew.bat -Pspec=java assemble
:libraries:couchbase-lite-java-core:compileJava
warning: [options] bootstrap class path not set in conjunction with -source 1.6
Note: Some input files use or override a deprecated API.
Note: Recompile with -Xlint:deprecation for details.
Note: Some input files use unchecked or unsafe operations.
Note: Recompile with -Xlint:unchecked for details.
1 warning
:libraries:couchbase-lite-java-core:processResources UP-TO-DATE
:libraries:couchbase-lite-java-core:classes
:libraries:couchbase-lite-java-core:jar
:compileJava
Note: Some input files use unchecked or unsafe operations.
Note: Recompile with -Xlint:unchecked for details.
:processResources UP-TO-DATE
:classes
:generateForestDBStoreJniHeaders
:generateCBForestJniHeaders
:compileWindows_amd64CouchbaseLiteJavaForestDBSharedLibraryCouchbaseLiteJavaFore
stDBCpp
Collatable.cc
C:\Program Files (x86)\Microsoft Visual Studio 12.0\VC\include\xlocale(337) : wa
rning C4530: C++ exception handler used, but unwind semantics are not enabled. S
pecify /EHsc
C:\couchbase-lite-java-forestdb\vendor\cbforest\CBForest\Collatable.cc(77) : err
or C2039: 'max' : is not a member of 'std'
C:\couchbase-lite-java-forestdb\vendor\cbforest\CBForest\Collatable.cc(77) : err
or C3861: 'max': identifier not found
C:\couchbase-lite-java-forestdb\vendor\cbforest\CBForest\Collatable.cc(141) : wa
rning C4068: unknown pragma
C:\couchbase-lite-java-forestdb\vendor\cbforest\CBForest\Collatable.cc(375) : wa
rning C4068: unknown pragma

slice.cc
C:\Program Files (x86)\Microsoft Visual Studio 12.0\VC\include\xlocale(337) : wa
rning C4530: C++ exception handler used, but unwind semantics are not enabled. S
pecify /EHsc
C:\couchbase-lite-java-forestdb\vendor\cbforest\CBForest\slice.cc(25) : error C2
059: syntax error : ':'
C:\couchbase-lite-java-forestdb\vendor\cbforest\CBForest\slice.cc(27) : error C2
059: syntax error : ':'

:compileWindows_amd64CouchbaseLiteJavaForestDBSharedLibraryCouchbaseLiteJavaFore
stDBCpp                                                                        F
AILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':compileWindows_amd64CouchbaseLiteJavaForestDBSharedL
ibraryCouchbaseLiteJavaForestDBCpp'.
> Multiple build operations failed.
      C++ compiler failed while compiling Collatable.cc.
      C++ compiler failed while compiling slice.cc.
  See the complete log at: file:///C:/couchbase-lite-java-forestdb/build/tmp/com
pileWindows_amd64CouchbaseLiteJavaForestDBSharedLibraryCouchbaseLiteJavaForestDB
Cpp/output.txt

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug
option to get more log output.

BUILD FAILED

Total time: 32.578 secs

C:\couchbase-lite-java-forestdb>
```

--- a/build-java.gradle
+++ b/build-java.gradle
@@ -422,7 +422,7 @@ model {
                 } else if (targetPlatform.operatingSystem.windows) {
                     cppCompiler.args "-I${org.gradle.internal.jvm.Jvm.current().javaHome}/include"
                     cppCompiler.args "-I${org.gradle.internal.jvm.Jvm.current().javaHome}/include/win32"
-                    linker.args 'Ws2_32.lib'
+                    linker.args 'Ws2_32.lib', 'Advapi32.lib', 'Gdi32.lib', 'User32.lib'
                 }
             }
         }

